edb/server/protocol/auth_ext/ui/components.py

```diff
@@ -37,14 +37,17 @@
 ]
 
 
+DEFAULT_BRAND_COLOR = "1f8aed"
```

Comment:
Oddly, the UI uses this as the default color, but emails used `007bff`, or at least _tried_ to use that color. I thought since the fallback never worked, it would be better to standardize on the one color we've actually successfully used 😅